src/doom/g_game.c

```diff
@@ -2943,7 +2943,7 @@ void G_TimeDemo (char* name)
 =================== 
 */ 
  
-boolean G_CheckDemoStatus (void) 
+boolean G_CheckDemoStatus (ticcmd_t* cmd) 
```

Comment:
This needs a `[crispy]` comment, please.

Reply:
Done